docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -0,0 +1,38 @@
+.. _tutorial_create_expectations_between_tables:
+
+Create Expectations Between Tables
+===================================
+
+This tutorial covers the workflow of creating and editing expectations that relate to data stored in different tables or datasets.
```

Comment:
Capitalize and link "Expectations" on first reference.

Reply:
Updated